packages/url/src/filter-url-for-display.js

```diff
@@ -19,5 +19,26 @@ export function filterURLForDisplay( url ) {
 		return filteredURL.replace( '/', '' );
 	}
 
+	// Prettify image urls
+	const mediaRegexp = /([\w|:])*\.(?:jpg|jpeg|gif|png|svg)/g;
```

Comment:
Do we need to be matching this globally? Are we expecting multiple matches?

```suggestion
	const mediaRegexp = /([\w|:])*\.(?:jpg|jpeg|gif|png|svg)/;
```